batinstall.sh

```diff
@@ -10,7 +10,8 @@ source ~/.bat_vim/config/plugins.vim
 source ~/.bat_vim/config/binds.vim
 source ~/.bat_vim/config/neocomplcache.vim
 source ~/.bat_vim/config/neosnippet.vim
-source ~/.bat_vim/config/unite.vim' > ~/.vimrc
+source ~/.bat_vim/config/unite.vim
+set t_Co=256' > ~/.vimrc
```

Comment:
This is not the right place for adding a setting. Take a look at config/base.vim. Also, have you taken into account the consequences of this change to different types of consoles?

Let's wait for @daneryl opinions.